src/util/document.js

```diff
@@ -0,0 +1,29 @@
+import fetch from 'node-fetch';
+
+import { tryPromise } from './promise';
+
+function convertDocumentToBiopax(docId, baseUrl = '') {
+  let SERVER_URL = baseUrl + `/api/document/biopax/${docId}`;
+  let makeRequest = () => fetch(SERVER_URL);
```

Comment:
`node-fetch` won't differentiate a response based on HTTP status. So you could get garbage in the response. I made a `util/fetch.js` helper for this.